src/Surface.h

```diff
@@ -197,9 +197,12 @@ class Plane: public Surface {
   /** The coefficient for the linear term in y */
   double _B;
 
-  /** The constant offset */
+  /** The coefficient for the linear term in y */
```

Comment:
This comment mirrors the one above for `_B` - I think this should be "linear term in z"